install-calico.tf

```diff
@@ -1,18 +1,16 @@
 resource "null_resource" "calico" {
-  count = "${var.calico_enabled ? 1 : 0}"
+  count = var.calico_enabled ? 1 : 0
 
   connection {
-    host        = "${hcloud_server.master.*.ipv4_address}"
-    private_key = "${file(var.ssh_private_key)}"
+    host        = hcloud_server.master.*.ipv4_address
+    private_key = file(var.ssh_private_key)
   }
 
   provisioner "remote-exec" {
-    inline = <<EOF
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/etcd.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/calico.yaml
-    EOF
+    inline = ["kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml"]
```

Comment:
```suggestion
    inline = [ "kubectl apply -f https://docs.projectcalico.org/v3.2/gettingtarted/kubernetes/installation/hosted/etcd.yaml", "kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml","kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/calico.yaml"]
```

Reply:
Okay, reverted the change and moved it into #13